### ISSUE TITLE
It shows "Please update the Results sheet with your Selection. Re-run program after your selections" when generate dynamic links is selected.

### ISSUE THREAD

**Issue by samyadh** [PathriK/GaanaDownloader-Extension#2]
![screen shot 2017-07-04 at 3 39 15 pm](https://user-images.githubusercontent.com/9761477/27826003-1f684236-60cf-11e7-8d0d-03946ea9b485.png)

**Comment by PathriK** [comment]
Did you do a search using the second option first?

**Comment by samyadh** [comment]
Yes i did i did search for one song and it was added in the CSV too.

**Comment by PathriK** [comment]
Can u send me the csvv and log files??

**Comment by samyadh** [comment]
![screen shot 2017-07-05 at 10 58 22 pm](https://user-images.githubusercontent.com/9761477/27876724-abad0946-61d5-11e7-9f96-71d41d1e97dc.png)
csv is not supported so added screenshot.

**Comment by PathriK** [comment]
Ok..the issue I think is... You need to add 'Yes' in the yesno column when
it asks to update the sheet . Then continue the script. That should make it
work I guess.

Alternatively can you try using the GaanaDownloader Chrome extension..and
let me know if u are facing any issue in it?

Thanks ,
Keerthi Kumar P

**Comment by samyadh** [comment]
![screen shot 2017-07-06 at 12 48 41 pm](https://user-images.githubusercontent.com/9761477/27899877-a6a922b2-6249-11e7-99d1-caac0eee265d.png)
Now an exception pops up.
Using extension nothing happens when clicked on download button.

**Comment by samyadh** [comment]
![screen shot 2017-07-06 at 12 54 49 pm](https://user-images.githubusercontent.com/9761477/27900097-83d8009a-624a-11e7-8b00-bbd21a6c6ba6.png)
Nothing happens if i click download after selecting album or song.

**Comment by PathriK** [comment]
Can you ping me in hangouts?? Let's take this off the GitHub issue
comments. It will be easy for us to discuss 😊

**Comment by PathriK** [comment]
Sorry, have been busy this week and was replying to you through my mobile only. So was not able to see the screenshots clearly. Now I see that you were using a Mac. Unfortunately, only windows is supported rite now. Will see how I can support Mac and Linux this weekend. 

Will let you know when I have a solution so that we can test it in your Mac ;)

**Comment by samyadh** [comment]
Awesome :)
